market/connector/uipath/meta.json

```diff
@@ -6,22 +6,16 @@
 		{
 			"id": "project-resource",
 			"data": {
-				"resources": [
-					{
-						"source": "demo/uiPathDemo/src/com/uipath/orchestrator/feature/TenantHeaderFeature.java",
-						"path": "src/com/uipath/orchestrator/feature/TenantHeaderFeature.java"
-					},
-					{
-						"source": "demo/uiPathDemo/src/com/uipath/orchestrator/feature/OAuth2Feature.java",
-						"path": "src/com/uipath/orchestrator/feature/OAuth2Feature.java"
-					},
-					{
-						"source": "demo/uiPathDemo/src/com/uipath/orchestrator/feature/FeatureConfig.java",
-						"path": "src/com/uipath/orchestrator/feature/FeatureConfig.java"
-					},
+				"directories": [
 					{
-						"source": "demo/uiPathDemo/src/com/uipath/orchestrator/feature/TokenStore.java",
-						"path": "src/com/uipath/orchestrator/feature/TokenStore.java"
+						"source": "demo/uiPathDemo/src/com/uipath/orchestrator/feature",
+						"includes": [ 
+							"TenantHeaderFeature.java", 
```

Comment:
I still use these UiPath features (e.g. TenantHeader) @ivy-lli  : 
but the copy mechanism has become easier -> now we can declare full directories as shown here.